turbo/rpchelper/helper.go

```diff
@@ -41,21 +41,23 @@ func _GetBlockNumber(requireCanonical bool, blockNrOrHash rpc.BlockNumberOrHash,
 	hash, ok = blockNrOrHash.Hash()
 	if !ok {
 		number := *blockNrOrHash.BlockNumber
-		if number == rpc.LatestBlockNumber {
+		switch number {
+		case rpc.LatestBlockNumber:
 			blockNumber = latestBlockNumber
```

Comment:
You probably want to make it consistent with `getLatestBlockNumber()` function in another file. Probably makes sense to move all these functions to `rpchelper.go` out of `cmd/rpcdaemon/commands` package